.gitignore

```diff
@@ -8,6 +8,7 @@
 !src
 src/*
 !src/Copy
+!src/styles
```

Comment:
So would the best route be to include all of the folder but exclude the layout file, or individually include the icons, theme and 'fonts' dir? Am going the second route right now just in case but will defer to your judgement